### ISSUE TITLE
Corner line colour

### ISSUE THREAD

**Issue by lkylove23** [buffylab/buffy-issues#7]
Hi

Some color has corner line colour while others don`t have

Please compare two dark colours

![image](https://cloud.githubusercontent.com/assets/3600194/17470841/a072c6f2-5d7a-11e6-95e7-1c8b2adcbafd.png)

Is there any option for it?

I prefer corner line without colour

Sincerely

KY Daniel Lee

**Comment by pawsong** [comment]
Right, I admit that edge highlighting is not working correctly :) It's due to my bad 3D skill and might take several days for fixing it :( I will investigate this bug this weekend!

Turning off the corner line (edge) highlight should be an option. I will add that option and let you know how to change it!

As always, thank you.

**Comment by lkylove23** [comment]
Thanks

I will check it later in Army 

ㅠㅠ

**Comment by pawsong** [comment]
@lkylove23 Okay :) My todolist for August is filled with your requests. Buffy will have been changed a lot when you come back later!

Please take care, Daniel.

**Comment by pawsong** [comment]
![swireframe](https://cloud.githubusercontent.com/assets/2945463/17509478/0bbb61c8-5e55-11e6-9087-f603fc837660.jpg)

Now you can turn off the wireframe (corner lines)! I hope this can help you :)
